src/makeRobotAccountant.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  return function(a) {
+    return function(b) {
+      count++;
+
+      if (count <= 3 || count % 2 !== 0) {
+        const sum = a + b;
+
+        return sum;
+      } else {
+        return 'Bzzz... Error!';
+      }
+    };
+  };
```

Comment:
[CODE STYLE] - You don't need to name functions if you return them right away. Consider using anonymous functions instead.